src/cli.ts

```diff
@@ -132,4 +133,43 @@ yargs(hideBin(process.argv))
       );
     }
   )
+  .command(
+    'compute-dapp-id',
+    'Computes the dApp ID for a given dApp alias and chain ID',
+    {
+      'dapp-alias': dappAlias,
+      'chain-id': chainId,
```

Comment:
Is this command public-facing or internal-facing? The rule is internal-facing interfaces use chain alias (because it's convenient and readable) and public-facing interfaces use chain ID (because the public don't know the aliases)

Reply:
Public. I also want to document it [here](https://docs.api3.org/oev-searchers/in-depth/data-feeds/#dapp-ids).

Reply:
I think the `unsafeComputeDappId` is sufficient for that purpose. The CLI is there only so that you don't have to bother with creating any script if you just want to quickly verify that.